app/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
     compileSdkVersion 31
     defaultConfig {
         applicationId "org.hackillinois.android"
-        minSdkVersion 21
+        minSdkVersion 23
```

Comment:
Now we're only supporting 7 year old phones instead of 8 :(

Reply:
the app will no longer run on my fridge :(